websites/O/Odysee/dist/metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+  "author": {
+    "name": "lunax",
+    "id": "251801168689364992"
+  },
+  "service": "Odysee",
+  "altnames": ["LBRY"],
+  "description": {
+    "en": "Web frontend for LBRY. LBRY does to publishing, what Bitcoin did to money."
+  },
+  "url": "odysee.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/GEL8nqd.png",
+  "thumbnail": "https://i.imgur.com/NQjEv5C.png",
```

Comment:
thumbnail should be a promotional banner or a screenshot of the website homepage.